assets/rich_text_components_definitions.js

```diff
@@ -45,7 +45,7 @@ var richTextComponents = {
     "frontend_id": "image",
     "tooltip": "Insert image",
     "icon_data_url": "/rich_text_components/Image/Image.png",
-    "preview_url_template": "/imagehandler/<[explorationId]>/<[filepath]>",
+    "preview_url_template": "/imagehandler/<[explorationId]>/<[filepath.name]>",
```

Comment:
Now the filepath is an object of the form
{
name: filename,
width: width,
height: height
}
So the preview url has been changed to <[filepath.name]>